exercicios/para-casa/testBiblioteca.py

```diff
@@ -0,0 +1,80 @@
+from unittest import TestCase 
+from BibliotecaP import Biblioteca 
+from Livro import Livro
+class testBiblioteca(TestCase):
+    def setUp(self):
+        self.biblioteca = Biblioteca()
+    def test_init_sucess(self):  #o python espera que o def comece com "test"
+        #arrange/act- chamar o cod fonte
+        biblioteca = Biblioteca()
+        #assert
+        self.assertIsInstance(self.biblioteca.livros, list)
+    
+    def test_adicionar_livro_sucess(self):
+        # Arrange 
+        #biblioteca = Biblioteca()
+        titulo = "mecânica quântica"
+        autor = "Feyman"
+        livro = Livro(titulo, autor)
+
+        # Act
+        self.biblioteca.adicionar_livro(livro)
+
+        #assert
+        self.assertEqual(1,len(self.biblioteca.livros))
+    
+    def test_add_livro_nao_numero_sucess(self):
+        #arrange
+        livro = 1997
+        #assert/act
+        with self.assertRaises(TypeError):
+            self.biblioteca.adicionar_livro(livro)
+
+    def test_listar_vazia(self):
+       #arrange
+       #lista = Biblioteca() foi retirado pois usamos o comando setUp no inicio
+       lista = Biblioteca()
+       #act
+       acervo_listados = lista.listar_livros()
+       self.assertEqual (0, len(acervo_listados)) == 0        
```

Comment:
Aqui não precisaria fazer a verificação com 0 no final, o assertEqual no fim das contas é um nome bonito para uma comparação `== `
Bastaria usar:
`self.assertEqual (0, len(acervo_listados))`

Se apertar control e for clicando com o botão direito no assertEqual e nos metodos dentro dele vai ver que na base ele faz essa comparação 

![image](https://github.com/reprograma/on26-python-s08-projeto-guiado-II/assets/15734807/cfe780b4-4241-4885-a566-62d17097b0e5)

E na [documentação](https://docs.python.org/3/library/unittest.html#assert-methods) também tem uma tabelinha explicando
![image](https://github.com/reprograma/on26-python-s08-projeto-guiado-II/assets/15734807/aea42691-330d-402b-9ca4-9540d3316dcf)